src/js/utils/obj.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file obj.js
+ */
+
+/**
+ * Array-like iteration for objects.
+ *
+ * @param  {Object}   object
+ * @param  {Function} fn
+ *         A callback function which is called for each key in the object. It
+ *         receives the value and key as arguments.
+ */
+export function each(object, fn) {
```

Comment:
I think it'd be a lot more. I'm willing to check to see how much more, but I suspect it's quite a bit (comparatively).